packages/web3-eth-accounts/src/errors.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable max-classes-per-file */
+
+export abstract class Web3Error extends Error {
```

Comment:
`Web3Error` should be imported from [web3-common](https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-common/src/errors.ts#L45) and not re-declared